docs/content/installation.rst

```diff
@@ -30,16 +30,20 @@ Installation
 	    :width: 50%
 
 
-.. note:: The assumption is made here that Anaconda has been installed in the default directory
+.. attention:: The assumption is made that Anaconda has been installed in the default directory:
+        ``%USERPROFILE%\anaconda3``
```

Comment:
do you really need the full Anaconda, or would it work with miniconda?

Reply:
I should try on my computer with `C:\Users\sebastienh\AppData\Local\Continuum\miniconda3` ...

Reply:
would be great. And we could try with Miniconda for sure. It would be a change for current users though